README.md

```diff
@@ -84,8 +84,8 @@ in a single realm include:
 * sub-realm sandboxes ([SES][ses] and [LavaMoat][lava-moat]) that virtualize
   evaluating guest modules and limit access to globals and built-in modules.
   This proposal prepares for the SES proposal to introduce `lockdown`, which
-  isolates all evaluators, including `eval`, `Function`, and this `Compartment`
-  module evaluator. That proposal will introduce the concern of per-compartment
+  isolates all evaluators, including `eval`, `Function`, and this `Loader`
```

Comment:
looks like you need a rebase 🤔

Reply:
I think, by rebasing, I’ve confused Github. I’ll iron it out.

Reply:
Ironed out. Thanks.